specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2022-12-01-preview/elasticsan.json

```diff
@@ -1960,6 +2146,109 @@
         "id"
       ],
       "description": "Virtual Network rule."
+    },
+    "PrivateEndpointConnectionList": {
+      "type": "object",
+      "description": "List of Private Endpoints",
+      "additionalProperties": false,
+      "properties": {
+        "value": {
+          "description": "An array of Private Endpoint Connection objects.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          }
+        },
+        "nextLink": {
+          "description": "URI to fetch the next section of the paginated response.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "PrivateEndpointConnection": {
+      "type": "object",
+      "description": " Response for PrivateEndpoint Connection object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Private Endpoint Connection Properties.",
+          "$ref": "#/definitions/PrivateEndpointConnectionProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ]
+    },
+    "PrivateEndpointConnectionProperties": {
+      "type": "object",
+      "description": " Response for PrivateEndpoint connection properties",
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning State of Private Endpoint connection resource",
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "privateEndpoint": {
+          "description": "Private Endpoint resource",
+          "$ref": "#/definitions/PrivateEndpoint"
+        },
+        "privateLinkServiceConnectionState": {
```

Comment:
this looks should be readonly

Reply:
kept it as per private-link.json provided from common-types.

Reply:
Why not use the one from common-types?